srcpkgs/python3-subprocess-tee/template

```diff
@@ -15,7 +15,7 @@ checksum=ff5cced589a4b8ac973276ca1ba21bb6e3de600cde11a69947ff51f696efd577
 make_check=no
 
 post_patch() {
-	vsed -e "/setuptools_scm_git_archive/d" -i pyproject.toml
+	vsed -e "/setuptools_scm_git_archive/d" -e "/pip/d" -i pyproject.toml
```

Comment:
Holy what the cow. Who on earth specifies that pip needs to be installed inside the virtualenv that pip creates for building?